samples/AzureFunctionsApp/Entities/Counter.cs

```diff
@@ -125,17 +102,27 @@ public static class CounterHelpers
     {
         _ = int.TryParse(request.Query["value"], out int value);
 
-        string name = "counter";
-
-        // switch to StateCounter if ?mode=1 or ?mode=state is supplied.
+        // switch to Counter_State if ?mode=1 or ?mode=state is supplied.
+        // or to Counter_Alt if ?mode=2 or ?mode=static is supplied.
+        string name;
         string? mode = request.Query["mode"];
-        if (int.TryParse(mode, out int m) && m == 1)
+        if (int.TryParse(mode, out int m))
```

Comment:
This is a little confusing to me since I'm not sure (as a reader of samples) whether "mode" is some concept that's important for durable entities.  It looks like we're using it to specify which entity type to use. Any reason not to just accept a string value which is the entity type name directly? That would make the intent of this method more clear, I think.

Reply:
it is either/or:

mode=0 or mode=entity
mode=1 or mode=state
mode=2 or mode=static

I'll expand on the comments for this.